api/config.py

```diff
@@ -85,6 +85,22 @@ class LocalConfig(BaseConfig):
     SQLALCHEMY_ECHO = False
     SECRET_KEY = "pancakes"
 
+    DB_ENGINE = os.getenv('DATABASE_ENGINE', 'mysql')
```

Comment:
When you rebase (you can rebase to the sjrumsby/sprint-with-us branch), this change will no longer be necessary. Adam beat you to the target branch, so his change got there first.

Reply:
Actually, even easier. Since you properly (good job!) split this change into it's own commit you can just drop it from your history.